beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -419,6 +419,7 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
     /// ## Errors
     ///
     /// May return a database error.
+    #[allow(unused)]
```

Comment:
This was no longer used after I refactored attestation processing, but it was added so recently I wasn't sure if I should delete it...